src/pages/[...slug].tsx

```diff
@@ -1,110 +1,75 @@
-import {
```

Comment:
This follows the example at https://shadcn.notion.site/next-drupal-query-bdbe85efca244b33b7b253ef87938826, with some modifications for this proposed structure.